lego/apps/users/serializers/users.py

```diff
@@ -106,19 +106,29 @@ class AdministrateUserSerializer(PublicUserSerializer):
     """
 
     class Meta(PublicUserSerializer.Meta):
-        fields = PublicUserSerializer.Meta.fields + (  # type: ignore
-            "abakus_groups",
-            "allergies",
-        )
+        fields = PublicUserSerializer.Meta.fields + ("abakus_groups",)  # type: ignore
 
 
-class AdministrateUserExportSerializer(PublicUserSerializer):
+class AdministrateUserAllergiesExportSerializer(PublicUserSerializer):
```

Comment:
Why are there two serializers with `allergies` now? Does it not make more sense to keep `AdministrateUserExportSerializer`, and simply extend that to be `AdministrateUserAllergiesSerializer`?

Reply:
The point is that when getting the allergies, we dont need the information that the export serializer gives (email and phone number). So the point was to simply send as little information as needed for those different scenarios

Reply:
But as it's now, you can access allergies through `EventAdministrateAllergiesExportSerializer` even if you're not the creator of the event?

And then, why change the name?

Reply:
Im changing back to normal now

Reply:
Actually I have to add 'allergies' back to the export serializer because if not it is not possible to get allergies if use_contact_tracing is true. 

Is it best to let it still be called just AdministrateUserExportSerializer or is it better to change it to AdministrateUserAllergiesExportSerializer (as I did in the original commit) since it then "adds" on to the allergies serializer?